core/chains/evm/config/chain_specific_config_test.go

```diff
@@ -0,0 +1,187 @@
+
+}
+
+func parseChainSpecificDefaults(chainConfigs map[int64]chainSpecificConfigDefaultSet) [][]string {
+	orderedChainIDs := make([]int, 0)
+	for k := range chainSpecificConfigDefaultSets {
+		orderedChainIDs = append(orderedChainIDs, int(k))
+	}
+	sort.Ints(orderedChainIDs)
+
+	var matrix [][]string
+	for index, id := range orderedChainIDs {
+		settings := chainConfigs[int64(id)]
+		data := [][]string{
+			{"EVM_EIP1559_DYNAMIC_FEES", strconv.FormatBool(settings.eip1559DynamicFees)},
+			{"ETH_TX_REAPER_INTERVAL", settings.ethTxReaperInterval.String()},
+			{"ETH_TX_REAPER_THRESHOLD", settings.ethTxReaperThreshold.String()},
+			{"ETH_TX_RESEND_AFTER_THRESHOLD", settings.ethTxResendAfterThreshold.String()},
+			{"ETH_FINALITY_DEPTH", strconv.FormatUint(uint64(settings.finalityDepth), 10)},
+			{"ETH_GAS_BUMP_PERCENT", strconv.FormatUint(uint64(settings.gasBumpPercent), 10)},
+			{"ETH_GAS_BUMP_THRESHOLD", strconv.FormatUint(uint64(settings.gasBumpThreshold), 10)},
+			{"ETH_GAS_BUMP_TX_DEPTH", strconv.FormatUint(uint64(settings.gasBumpTxDepth), 10)},
+			{"ETH_GAS_BUMP_WEI", settings.gasBumpWei.String()},
+			{"EVM_GAS_FEE_CAP_DEFAULT", settings.gasFeeCapDefault.String()},
+			{"ETH_GAS_LIMIT_DEFAULT", strconv.FormatUint(uint64(settings.gasLimitDefault), 10)},
+			{"ETH_GAS_LIMIT_MULTIPLIER", strconv.FormatFloat(float64(settings.gasLimitMultiplier), 'f', 2, 64)},
+			{"ETH_GAS_LIMIT_TRANSFER", strconv.FormatUint(uint64(settings.gasLimitTransfer), 10)},
+			{"ETH_GAS_PRICE_DEFAULT", settings.gasPriceDefault.String()},
+			{"EVM_GAS_TIP_CAP_DEFAULT", settings.gasTipCapDefault.String()},
+			{"EVM_GAS_TIP_CAP_MINIMUM", settings.gasTipCapMinimum.String()},
+			{"ETH_HEAD_TRACKER_HISTORY_DEPTH", strconv.FormatUint(uint64(settings.headTrackerHistoryDepth), 10)},
+			{"ETH_HEAD_TRACKER_MAX_BUFFER_SIZE", strconv.FormatUint(uint64(settings.headTrackerMaxBufferSize), 10)},
+			{"ETH_HEAD_TRACKER_SAMPLING_INTERVAL", settings.headTrackerSamplingInterval.String()},
+			{"LINK_CONTRACT_ADDRESS", settings.linkContractAddress},
+			{"ETH_LOG_BACKFILL_BATCH_SIZE", strconv.FormatUint(uint64(settings.logBackfillBatchSize), 10)},
+			{"ETH_LOG_POLL_INTERVAL", settings.logPollInterval.String()},
+			{"ETH_MAX_GAS_PRICE_WEI", settings.maxGasPriceWei.String()},
+			{"ETH_MAX_IN_FLIGHT_TRANSACTIONS", strconv.FormatUint(uint64(settings.maxInFlightTransactions), 10)},
+			{"ETH_MAX_QUEUED_TRANSACTIONS", strconv.FormatUint(uint64(settings.maxQueuedTransactions), 10)},
+			{"ETH_MIN_GAS_PRICE_WEI", settings.minGasPriceWei.String()},
+			{"ETH_USE_FORWARDERS", strconv.FormatBool(settings.useForwarders)},
+			{"ETH_RPC_DEFAULT_BATCH_SIZE", strconv.FormatUint(uint64(settings.rpcDefaultBatchSize), 10)},
+			{"OCR_CONTRACT_TRANSMITTER_TRANSMIT_TIMEOUT", settings.ocrContractTransmitterTransmitTimeout.String()},
+			{"OCR_DATABASE_TIMEOUT", settings.ocrDatabaseTimeout.String()},
+			{"OCR_OBSERVATION_GRACE_PERIOD", settings.ocrObservationGracePeriod.String()},
```

Comment:
Why are we using legacy global env var names here? IMHO this is very misleading and confusing.